manifests/master/simp_auth.pp

```diff
@@ -1,55 +1,48 @@
 # Add SIMP-specific entries to PuppetServer's auth.conf
 #
+# For documentation about *_allow and *_deny, see the puppetserver docs
+# @see https://puppet.com/docs/puppetserver/5.2/config_file_auth.html#allow-allow-unauthenticated-and-deny
```

Comment:
we are delivering puppetserver 2.8.1 with the SIMP ISO, so why the reference to puppetserver 5.2?

Reply:
Good catch